README.md

```diff
@@ -789,3 +790,49 @@ Response:
 must provide file_list field 
 ```
 
+# Autodetect App and File Type IDs
+
+## App type IDs
+
+These are the currently supported upload app type IDs:
+
+```
+fastq_reads_interleaved
```

Comment:
How did you generate this list, is there something I can check against?

Reply:
It's from https://github.com/kbase/staging_service/blob/master/staging_service/autodetect/Mappings.py#L34-L53